deployment/assets/env/consumer_connector.env

```diff
@@ -0,0 +1,31 @@
+# control plane specific config
+EDC_IAM_ISSUER_ID="did:web:localhost%3A7083"
+WEB_HTTP_PORT=8080
+WEB_HTTP_PATH="/api"
+WEB_HTTP_MANAGEMENT_PORT=8081
+WEB_HTTP_MANAGEMENT_PATH="/api/management/"
+WEB_HTTP_PROTOCOL_PORT=8082
+WEB_HTTP_PROTOCOL_PATH="/api/dsp"
+WEB_HTTP_CONTROL_PORT=8083
+WEB_HTTP_CONTROL_PATH="/api/control"
+WEB_HTTP_CATALOG_PORT=8084
+WEB_HTTP_CATALOG_PATH="/api/catalog"
+WEB_HTTP_VERSION_PORT=8085
+WEB_HTTP_VERSION_PATH="/api/version"
+EDC_API_AUTH_KEY="password"
+EDC_IAM_DID_WEB_USE_HTTPS="false"
+EDC_IAM_STS_PRIVATEKEY_ALIAS="did:web:localhost%3A7083-alias"
+EDC_IAM_STS_PUBLICKEY_ID="did:web:localhost%3A7083#key-1"
+EDC_DSP_CALLBACK_ADDRESS="http://localhost:8082/api/dsp"
+EDC_PARTICIPANT_ID="did:web:localhost%3A7083"
+EDC_CATALOG_CACHE_EXECUTION_DELAY_SECONDS=5
+EDC_CATALOG_CACHE_EXECUTION_PERIOD_SECONDS=10
+EDC_MVD_PARTICIPANTS_LIST_FILE="deployment/assets/participants/participants.local.json"
+
+# dataplane specific config
+EDC_RUNTIME_ID = "consumer-embedded-runtime"
```

Comment:
Pls avoid spaces around = , or the env variables keys will have a space at the end and won't be found when looking up